Frontend/android-native/app/src/main/res/values/constants.xml

```diff
@@ -31,6 +31,7 @@
     <string name="nammayatripartner" translatable="false">nammayatripartner</string>
     <string name="LANGUAGE_KEY" translatable="false" >LANGUAGE_KEY</string>
     <string name="preference_file_key" translatable="false">namma_yatri_app_local_keys</string>
+    <string name="UPDATE_TYPE" translatable="false">updateType</string>
```

Comment:
if its storage key then define this in capital snake case

Reply:
also define this in `Storage`

Reply:
Updated the changes.